addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiAqaraSensorMagnetHandler.java

```diff
@@ -0,0 +1,97 @@
+package org.openhab.binding.mihome.handler;
```

Comment:
It seems that this file has a missing, outdated or otherwise non standard copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.